Sources/AblyAssetTrackingCore/Extension/GeoJSONMessage+Location.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+public extension GeoJSONMessage {
+    func toLocation() -> Location {
+        return Location(
+            coordinate: LocationCoordinate(latitude: geometry.latitude, longitude: geometry.longitude),
+            altitude: geometry.altitude,
+            ellipsoidalAltitude: .zero,
+            horizontalAccuracy: properties.accuracyHorizontal,
+            verticalAccuracy: properties.accuracyVertical ?? -1,
+            course: properties.bearing ?? -1,
+            courseAccuracy: properties.accuracyBearing ?? -1,
+            speed: properties.speed ?? -1,
+            speedAccuracy: properties.accuracySpeed ?? -1,
```

Comment:
Are you sure that those `-1` won't break anything? [On Android](https://github.com/ably/ably-asset-tracking-android/blob/main/publishing-sdk/src/main/java/com/ably/tracking/publisher/LocationHistoryModels.kt#L20-L37) we're using `0` as the default values :wink:

Reply:
to be honest I'm not 100% sure 😅  I'll change it for consistency

Reply:
269f1435c399760796aea6074d52b40324b8f5cd